src/main/java/auth/TdpRecruitmentPasswordStore.java

```diff
@@ -0,0 +1,194 @@
+package auth;
```

Comment:
we should keep the copyright notice, like [here](https://github.com/wojciechh-kainos/tdp-invest/blob/master/src/main/java/auth/TdpInvestPasswordStore.java)